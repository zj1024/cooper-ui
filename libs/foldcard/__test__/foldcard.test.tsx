import * as React from 'react'
import * as renderer from 'react-test-renderer'
import Foldcard from '../index'

describe('- Tooltip component', () => {
  it('exist', () => {
    const component = renderer.create(<Foldcard fold={'test'}>content</Foldcard>)
    let tree: any = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
