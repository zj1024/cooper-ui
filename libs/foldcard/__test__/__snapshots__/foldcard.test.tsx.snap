// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`- Tooltip component exist 1`] = `
<section
  className="coo-foldcard"
>
  <div
    className="coo-foldcard-content"
  >
    content
  </div>
  <footer
    className="coo-foldcard-footer"
  >
    <div
      className="coo-foldcard-footer-main"
    >
      <p
        className="coo-foldcard-footer-spread"
        onClick={[Function]}
      >
        <span
          className="coo-icon coo-foldcard-footer-icon"
          dangerouslySetInnerHTML={
            Object {
              "__html": undefined,
            }
          }
        />
        <span>
          展开
        </span>
      </p>
    </div>
  </footer>
  <div
    className="coo-foldcard-card coo-foldcard-card-hidden"
  >
    test
  </div>
</section>
`;
